docs/getting-started/setup/editor-types.md

```diff
@@ -0,0 +1,83 @@
+---
+category: setup
+menu-title: Editor types
+meta-title: Editor types | CKEditor 5 documentation
+meta-description: Learn about available editor types.
+order: 25
+modified_at: 2024-06-06
+---
+# Editor types
+
+The editor's user interface is mostly dependent on the editor types. Functionality of the editor is provided by specialized features, accessible via a configurable toolbar or keyboard shortcuts. Some of these features are only available with certain editor types.
+
+<info-box>
+	All of these elements can most easily be configured with [CKEditor&nbsp;5 Builder](https://ckeditor.com/ckeditor-5/builder?redirect=docs).
+</info-box>
+
+There are six ready-made editor types (see below) available for CKEditor&nbsp;5. They offer different functional approach to editing as well as different UI solutions. Editor types are imported from the main `ckeditor5` package, the same way features are imported, as shown in the {@link getting-started/quick-start Quick start} guide.
+
+Other, custom-taylored editor types can be made using the {@link framework/external-ui CKEditor&nbsp;5 Framework}.
+
+For example, this code will import the classic editor type and some basic text formatting plugins. It also provides configuration of the {@link getting-started/setup/toolbar main editor toolbar}.
+
+```js
+import { ClassicEditor, Bold, Italic, Link } from 'ckeditor5'; // Imports.
+
+ClassicEditor // Editor type declaration.
+	.create( document.querySelector( '#editor' ), {
+		plugins: [ Bold, Italic, Link ], // Plugins import.
+		toolbar: [ 'bold', 'italic', 'link' ] // Toolbar configuration.
+	} )
+	.then( /* ... */ )
+	.catch( /* ... */ );
+```
+
+## Classic editor
+
+Classic editor is what most users traditionally learnt to associate with a rich-text editor &ndash; a toolbar with an editing area placed in a specific position on the page, usually as a part of a form that you use to submit some content to the server.
+
+{@img assets/img/editor-type-classic.png 800 Classic editor type.}
+
+See an {@link examples/builds/classic-editor example of the classic editor} in action.
+
+## Inline editor
+
+Inline editor comes with a floating toolbar that becomes visible when the editor is focused (for example, by clicking it). A common scenario for using inline editor is offering users the possibility to edit content (such as headings and other small areas) in its real location on a web page instead of doing it in a separate administration section.
```

Comment:
```suggestion
The inline editor comes with a floating toolbar that becomes visible when the editor is focused (for example, by clicking it). A common scenario for using the inline editor is offering users the possibility to edit content (such as headings and other small areas) in its real location on a web page instead of doing it in a separate administration section.
```

Reply:
Maybe we should unify whether we want to use articles before editor types.